Новый корабль сепаратистов Элизиума

NofearNr opened the pull request:
Да... он работает на Тесле...дя-дя не надо в щитспавн. Стоит отметить, что на локалке я его не тестил(я пытался честно, я отдельно пробовал скачивать билд, но все равно когда запускаешь, нажимаешь join game оно все белыми квадратиками, я уже и кэш очищал, и уже обновил бьонд до 15 или какой там версии, ну ваще ничё его не берет), ошибок по сути, быть не должно. Точно не должно. Жду вещи зеленых

NofearNr committed:
Новый корабль сепаратистов Элизиума

Да... он работает на Тесле...дя-дя не надо в щитспавн. Стоит отметить, что на локалке я его не тестил(я пытался честно, я отдельно пробовал скачивать билд, но все равно когда запускаешь, нажимаешь join game оно все белыми квадратиками, я уже и кэш очищал, и уже обновил бьонд до 15 или какой там версии, ну ваще ничё его не берет), ошибок по сути, быть не должно. Точно не должно. Жду вещи зеленых

NofearNr committed:
Небольшой fix Алдаама

Добавление кнопочек для створок

molnizz commented:
Не одобрено, пойдёт на переделку

molnizz reviewed (changes requested):
Переделать описание, убрать непереносные танки с воздухом

NofearNr committed:
Изменение описания и добавления ткомов кораблю Алдаама

Добавление комнаты телекоммуникаций, + коннектор с аир помпой, также в зоопарк сепаров добавлен один интересный экземпляр

NofearNr commented:
Переделал

MrCat15352 commented:
@molnizz пока ты не апрувнешь своё ревью, этот ПР так и будет весеть тут

NofearNr committed:
Кэп, это последний!

В общем, теперь то точно все. Убрал ту комнатку которую я оставлял для отката, и добавил в разных местах фаер алярмы и локи

NofearNr committed:
Кэп.... я устал драить эту чертову палубу!!!!

Ну да, тут нет слов. Смещение корабля и размеров карты

NofearNr committed:
Капитан.... я умираю от удушья.... дайте водички...

Душним-душним. Зоны которые должны быть в космосе добавлены...

molnizz reviewed (approved):
Иншалах
